spec/types/intersection.spec.ts

```diff
@@ -31,4 +31,39 @@ describe('intersection', () => {
       }
     );
   });
+
+  it('supports nullable intersection types', () => {
```

Comment:
Thank you! :bowing_man:  Those were not working before, were they?

Reply:
They were not!